debian/not-installed

```diff
@@ -1,7 +1,6 @@
 lib/systemd/system/mariadb.service # Installed by rules file
 lib/systemd/system/mariadb@.service # Installed by rules file
 usr/bin/mysql_config # We already have the MariaDB variant
-usr/bin/mysql_embedded # Huge 500 MB file. Not intended for distribution via any disto package.
 usr/bin/mytop # Mytop is distributed from a separate source package
 usr/bin/sst_dump # Use the one from rocksdb-tools package
 usr/lib/sysusers.d/sysusers.conf
```

Comment:
why is sysusers.conf and tmpfile.conf not installed? Sure they need to be be mariadb.conf in the respective directories for naming. Any other reason?

Reply:
Actually in 742c36d048e0 you can configure exactly where sysusers.conf (named mariadb.conf) actually land. make install should put them in the right place. @ottok would this work for debian? That was my intent when merging that patch.

Reply:
This commit does not change that. I'll post separately on my init/systemd changes in another PR.